NetworkChatClient/src/main/java/ru/gb/java2/chat/client/controllers/Controller.java

```diff
@@ -0,0 +1,139 @@
+            return;
+        }
+        String sender = application.getCurrentUserName();
+        String receiver;
+        if (!userList.getSelectionModel().isEmpty()) {
+            receiver = userList.getSelectionModel().getSelectedItem().toString();
+        } else {receiver = currentUserLabel.getText();}
+        if(receiver.equals(TO_ALL_USERS)) {
+            message = String.join(",",sender,receiver, message);
+        } else {
+            message = String.join(",", USER_PRIVATE_MESSAGE,sender, receiver, message);
+        }
+        try {
+            network.sendMessage(message);
+        } catch (IOException e) {
+            application.showNetworkErrorDialog("Ошибка передачи данных по сети", "Не удалось отправить сообщение!");
+        }
+        chatFieldAddMessage("Я -> " + receiver, messageField.getText().trim());
+
+    }
+
+    private void chatFieldAddMessage(String sender, String message) {
+        chatField.appendText(DateFormat.getDateTimeInstance().format(new Date()));
+        chatField.appendText(System.lineSeparator());
+        if (sender != null) {
+            chatField.appendText(sender + ":");
+            chatField.appendText(System.lineSeparator());
+        }
+        chatField.appendText(message);
+        chatField.appendText(System.lineSeparator());
+        chatField.appendText(System.lineSeparator());
+        messageField.clear();
+    }
+
+    @FXML
+    private void choiceUser(MouseEvent mouseEvent) {
+        currentUserLabel.setText(userList.getSelectionModel().getSelectedItem().toString());
+        privateUser = userList.getSelectionModel().getSelectedItem().toString();
+    }
+
+    public void messageFieldKeyPressed(KeyEvent keyEvent) {
+        messageField.setOnKeyPressed(event -> {
+            if (event.getCode() == KeyCode.ENTER) {
+                event.consume();
+                if (event.isShiftDown() || event.isControlDown() || event.isAltDown()) {
+                    messageField.appendText(System.lineSeparator());
+                } else {
+                    if(!messageField.getText().isEmpty()){
+                        sendMessageAction();
+                    }
+                }
+            }
+        });
+    }
+    public void setNetwork(Network network) {
+        this.network = network;
+        network.waitMessages(message -> Platform.runLater(() -> {
+            if(message.startsWith(USER_LIST_PREFFIX)) {
+                userListUpdate(message);
+            } else if(message.startsWith(USER_PRIVATE_MESSAGE)) {
+                privateMessageProcess(message);
+            } else {
+                allUsersMessage(message);
+            }
+        }));
+    }
+
+    private void allUsersMessage(String message) {
+        String[] tempMessage = message.split(",");
+        chatFieldAddMessage(tempMessage[0] + " -> " + tempMessage[1], tempMessage[2]);
+    }
+
+    private void privateMessageProcess(String message) {
+        String[] tempMessage = message.split(",");
```

Comment:
```suggestion
        String[] tempMessage = message.split(",", 3);
```
иначе не сможем корректно обработать сообщение с запятыми